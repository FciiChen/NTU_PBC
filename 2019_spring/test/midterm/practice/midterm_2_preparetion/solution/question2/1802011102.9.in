VGT, WkT,WTx5uKFo8,xSDh0t78,Jhr
BREAK
VGT ,  G4WpU4kg
BREAK
