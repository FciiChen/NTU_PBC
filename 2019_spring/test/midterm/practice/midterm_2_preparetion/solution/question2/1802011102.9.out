VGT        WkT        WTx5uKFo8  xSDh0t78   Jhr        G4WpU4kg  
